_pages/attributes.md

```diff
@@ -6,13 +6,158 @@ lead: >
 
 Here are the possible attributes that can be requested at a given IAL. This table contains the available user attributes, the IAL they are associated with, and how they can be accessed in OpenID Connect and SAML.
 
-| Attribute | IAL1 | IAL2 | OpenID Connect | SAML |
```

Comment:
No love for markdown tables here

Reply:
It looks like the `verified_at` attribute did not actually get added to this table. Was that intentional?

Reply:
Well I do like Markdown tables a lot but this was getting tough to read and edit.

And yes I forgot to add the attribute after I converted it :see_no_evil:

Reply:
Fixed in 491c75a50f869fea1ce00d6bfefc25639d837dd6